functions/zram.bash

```diff
@@ -0,0 +1,38 @@
+check_zram_mounts() {
+  local FILE=/etc/ztab
+  local i=0
+
+  if ! [ -f ${FILE} ]; then
+    echo "ZRAM not installed - test successful."
+    return 0
+  fi
+
+  while read -r line; do
+    case "$line" in
+      "#"*) continue ;;
+      "")   continue ;;
+      *)    set -- $line
+            TYPE=$1
+            if [ "${TYPE}" == "swap" ]; then
+              if [ -z "`/sbin/swapon | /bin/grep zram`" ]; then
+                echo "`basename $0` error: swap not on zram"
+                return 1
+              fi
+            else
+              OVERLAY=$5
+              LOWER=$6
```

Comment:
Maybe good idea to check OVERAY and LOWER before using it on the commnd line...

Reply:
No. It isn't used via CLI but parses ztab file.

Reply:
Understood. I was more concerned about the content of the ztab file. Code injection does not seem to be a problem here. But in case ztab is broken, or the format changes over time, users won't not get a reasonable error message... 
btw: Can LOWER be removed or have I overlooked something?

Reply:
reworked